src/webview/static/base.css

```diff
@@ -142,6 +142,18 @@ body {
   padding: 0 16px 16px 80px;
 }
 
+.message-muted {
+  /* If a message is both brief and muted, we don't want to save the extra
+   * space for the avatar, since it's not shown anyways. */
+  padding: 16px;
```

Comment:
I don't think I follow what this comment is saying about "both brief and muted". Perhaps that'll end up being moot in a final version of this branch? I see this goes away in a later commit in this revision of the branch.

Reply:
Yeah, this goes away later.

A "brief" message is one that doesn't show the avatar or username (not a great name, but the one we use). This is so that we don't have the message about the fact that it's muted have a big margin on the left side, the way there normally is when there's no user avatar.

This goes away because I coalesce messages later, so brief messages never have any content in them.